pkg/virt-operator/resource/generate/rbac/exportproxy.go

```diff
@@ -79,6 +79,30 @@ func newExportProxyClusterRole() *rbacv1.ClusterRole {
 					"get", "list", "watch",
 				},
 			},
+			{
+				APIGroups: []string{
+					"apiextensions.k8s.io",
+				},
+				Resources: []string{
+					"customresourcedefinitions",
```

Comment:
Why do we need this?

Reply:
Thanks @xpivarc. This is currently required since the `SetupExportProxyTLS` function call needs to pass the `clusterConfig`. And the clusterConfig requires also the CRD. What I can do is to watch only the kubevirt resource, avoiding this rule to the required clusterrole. WDYT?

Reply:
Please check but I believe we don't require to watch crds in this case. It might be a good indicator that we can refactor these bits a little bit.

Reply:
I have already checked this. And yes it works. I will push the changes soon. Thanks